tools/scripts/gen-env-files.sh

```diff
@@ -445,6 +445,38 @@ MEDIA_STREAM_TIMEOUT=60000" >> ./services/audio/audio.env
     echo "./services/audio/audio.env created"
 fi
 
+## Audio - Clip Producer
+if test -f "./services/audio/audio.env"; then
```

Comment:
If the service has only one `.env` you don't need to name more than `.env`

Reply:
Fixed for audio and capture.